app/views/format_fotos/_form.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= form_for :format_foto, url: format_fotos_path do |f| %>
+  <p>
+    <%= f.label :name, 'Назва формату фото' %>
+    <%= f.text_field(:name, { class: 'form-control' }) %>
+  </p>
+  <p>
+    <br>
+    <%= f.submit('Зберегти', { class: 'btn btn-success' }) %>
+  </p>
+<% end %>
```

Comment:
@romigos https://github.com/rubocop-hq/ruby-style-guide#newline-eof 
and next places

Reply:
fixed